graphql/util.go

```diff
@@ -259,6 +259,17 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 		if version == nil {
 			return errors.Errorf("Couldn't find patch for id %v", p.Version), http.StatusInternalServerError, "", ""
 		}
+
+		v, err := model.VersionFindOneId(p.Version)
```

Comment:
We don't need to call this here; you can see at line 259 that we've already checked that the  version isn't nil (it's passed into schedule patch)

Reply:
done